easybuild/easyconfigs/a/Autotools/Autotools-20150119-intel-2015a.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'Bundle'
+
+name = 'Autotools'
+version = '20150119' # date of the most recent change
+
+homepage = 'http://autotools.io'
+description = """This bundle collect the standard GNU build tools: Autoconf, Automake and libtool"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
```

Comment:
no, this was a very deliberate choice. We want to compile this with the lowest common denominator. Performance etc doesn't come into play in this.